docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -139,9 +139,38 @@ All tests are executed until your test with a breakpoint is reached. Step throug
 
 ### Use Test Explorer
 
-`Microsoft.Testing.Platform` tests integrate with Visual Studio's **Test Explorer**. Tests can be run and debugged from **Test Explorer**.
+To run a test, navigate to **Test Explorer**, select the test (or tests) to run. Right select it, and choose **Run**. Similarly to debug a test, select the test (or tests) in **Test Explorer**, right select and choose **Debug**.
+
+> [!TIP]
+> Automatic update of tests without building the project isn't available.
+
+### [Visual Studio Code](#tab/visual-studio-code)
+
+The C# extension along with [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) allows you to debug/run tests in Visual Studio code, as well as adds integration with Visual Studio Code's Test Explorer.
+
+#### Run the app with Visual Studio
+
+`Microsoft.Testing.Platform` test project are built as executables, and can be run directly. This runs all the tests in the given executable, unless a filter is provided.
+
+1. Navigate to a test file that you want to run tests for.
+1. Use <kbd>Ctrl</kbd>+<kbd>F5</kbd> to run the selected test project. If you have multiple projects a popup will ask you to select the one to run.
 
-To enable running tests with MSTest runner, go to **Tools** > **Options** > **Preview Features** and select `Use testing platform server mode`.
```

Comment:
Is this enabled by default now?

Reply:
@drognanar You should also mention (can be in a separate PR) the feature flag for VS.

Reply:
the feature flag is enabled by default in the coming versions of VS/VsCode

Reply:
also, the feature flag won't currently show up in the Tools->Options

the idea is that you can still change whether or not to use the server mode by removing the TestingPlatformServer capability in csproj, but I'm not sure if we should add this to the documentation until we get feedback that there are issues with the server mode.